python/lib/sift_py/yaml/channel_test.py

```diff
@@ -0,0 +1,169 @@
+import pytest
+
+from sift_py.ingestion.config.yaml.error import YamlConfigError
+from sift_py.yaml import channel
+
+
+def test__validate_channel_anchor():
```

Comment:
I'm assuming it's because it's a private function, so it kept the extra underscore at the front of the channel name. I've seen this in a few places in the repo. I don't have strong feelings about it personally, in a way it does indicate to me as I'm reading that the function being tested is private without having to go check separately.